frontend/src/lib/components/InsightLegend/InsightLegendRow.tsx

```diff
@@ -53,24 +53,27 @@ export function InsightLegendRow({ rowIndex, item }: InsightLegendRowProps): JSX
         formatPropertyValueForDisplay
     )
 
+    const isPrevious = !!item.compare && item.compare_label === 'previous'
+    const adjustedIndex = isPrevious ? item.seriesIndex - totalItems / 2 : item.seriesIndex
```

Comment:
Am missing why we do `item.seriesIndex - totalItems / 2` for previous?
We already adjust the `baseHex` for previous + if I understand correctly, previous should have the lowest index out of all series items?

Might have missed something

Reply:
The reason for this is that the previous item should have the same color as the current item. This adjustment will make it have the same index and thus the same color.

The other adjustment on `baseHex` tones down opacity a bit, so that you can distinguish previous and total items.

This is best seen with a single series - notice how both items are the same blue color, just the previous series is a bit lighter:
<img width="377" alt="Screenshot 2024-07-23 at 08 10 51" src="https://github.com/user-attachments/assets/5b82c705-bb6b-430d-ae3a-646123ba1373">